src/types/router.ts

```diff
@@ -26,14 +26,18 @@ export type RouterOptions = {
    * Specifies the history mode for the router, such as 'hash', 'history', or 'abstract'.
    */
   historyMode?: RouterHistoryMode,
+  /**
+   * Specifies the desired host for the router, defaults to host of initialUrl.
+   */
+  host?: string,
```

Comment:
I don't feel strongly here but thinking if you can use this or initialUrl maybe we just use one? Fine as is. Just thinking out loud.

Reply:
yeah, I liked the explicit nature of having a separate prop but I agree that the fact that it's duplicative and potentially might disagree means we should probably kill it.